src/components/popper/PopperComponent.js

```diff
@@ -27,9 +27,11 @@ const PopperComponent = (props: Props) => {
             {isOpen && (
                 <Popper placement={popperPlacement} modifiers={modifiers}>
                     {({ ref, style, placement, scheduleUpdate }) => {
+                        const { style: contentStyles } = popperContent.props;
+                        const updatedStyles = { ...contentStyles, ...style };
```

Comment:
Minor, but not sure this variable is needed, since its value is used immediately and only once.